src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/Config/ConditionalConfig.cs

```diff
@@ -89,6 +89,19 @@ public static IReadOnlyList<IOperationProvider> ConditionalSetup(ConditionalType
                 case ConditionalType.JsxBlockComment:
                     setup = ConditionalBlockCommentConfig.GenerateConditionalSetup("{/*", "*/}");
                     break;
+                case ConditionalType.VB:
+                    setup = ConditionalLineCommentConfig.GenerateConditionalSetup("#", new ConditionalKeywords
+                    {
```

Comment:
This sets up '#' to be the line comment start token, so the conditionals will be #If, #ElseIf, etc. But SimpleConfigModel sets up single-quote as the flag-prefix character. This will work, just seems a bit odd to have different comment characters for the two situations.